kebechet/kebechet_runners.py

```diff
@@ -172,11 +172,27 @@ def run(
         return
 
     managers = config.managers
-
+    runtime_environments: List[str]
+    if config.config.get("overlays_dir"):
+        runtime_environments = [
+            e["name"] for e in config.config.get("runtime_environments")
+        ]
+    else:
+        runtime_environments = [config.config["runtime_environments"][0]["name"]]
     for manager in managers:
         # We do pops on dict, which changes it. Let's create a soft duplicate so if a user uses
         # YAML references, we do not break.
         manager = dict(manager)
+        disallow_run_times = manager.get("disallow", None)
+        allowed_run_times = manager.get("allow", None)
+        if disallow_run_times:
+            runtime_environments = list(
+                set(runtime_environments) - set(disallow_run_times)
+            )
+        if allowed_run_times:
+            runtime_environments = list(
+                set(runtime_environments).intersection(set(allowed_run_times))
+            )
```

Comment:
we should check on some earlier points to validate the schema and if the allow_run_time and disalow_run_times match the existing runtimes, if not open the issue for the user.

now for this code block:
will a simple if work here:
```
if allowed_run_times:
runtime_environments = allowed_run_time 
```